platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformCachingIntegrationTest.groovy

```diff
@@ -1084,7 +1084,7 @@ class ArtifactTransformCachingIntegrationTest extends AbstractHttpDependencyReso
 
     def "transform is executed in different workspace for different file produced in chain"() {
         given:
-        buildFile << declareAttributes() << withJarTasks() << duplicatorTransform << """
+        buildFile << declareAttributes() << withJarTasks() << duplicatorTransform(incremental: true) << """
```

Comment:
❓ Are we losing some coverage here?

Reply:
Added a test specifically for non-incremental transforms.